packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.component.tsx

```diff
@@ -13,15 +13,15 @@ import { ResourcesContext } from '../../../offline.resources';
 
 export const IdField: React.FC = () => {
   const { identifierTypes } = useContext(ResourcesContext);
-  const { setFieldValue, inEditMode } = useContext(PatientRegistrationContext);
+  const { values, setFieldValue, inEditMode } = useContext(PatientRegistrationContext);
```

Comment:
I think this work comes up again. It was implemented earlier, right?

Reply:
I'm not sure I understand the question?